tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5568,6 +5584,10 @@ declare namespace ts {
         textSpan: TextSpan;
         classificationType: ClassificationTypeNames;
     }
+    interface ClassifiedSpan2020 {
+        textSpan: TextSpan;
+        classificationType: number;
```

Comment:
This should definitely be not number but enum.. How does API user know what number means which classification otherwise?